queries/ruby/highlights.scm

```diff
@@ -39,32 +48,45 @@
 "defined?" @function
 
 (call
-  receiver: (constant) @constant)
-(method_call
-  receiver: (constant) @constant)
-(call
-  method: (identifier) @function)
-(method_call
-  method: (identifier) @function)
-(call
-  method: (constant) @function)
+  [
+   receiver: (constant) @constant
+   method: [
+            (identifier) @function
+            (constant) @function
+            ]
+   ])
+
 (method_call
-  method: (constant) @function)
+  [
+   receiver: (constant) @constant
+   method: [
+            (identifier) @function
+            (constant) @function
+            ]
```

Comment:
Isn't this refectorable even further in this ?
```suggestion
   method: [
            (identifier)
            (constant)
            ] @function
```

Reply:
I did a quick test and yes that works! I've updated accordingly